### ISSUE TITLE
Deal with non-UTF data in deferreds

### ISSUE THREAD

**Issue by squaremo** [jkcfg/jk#51]
Currently the result from deferreds is a flatbuffers `string`, i.e., UTF8. But we may want to have arbitrary binary data, and treat it as an ArrayBuffer (e.g., if we're doing any kind of checksumming or digests, we want the bytes).

**Comment by squaremo** [comment]
The `Data` response to a deferred now carries `[ubyte]`, which is the flatbuffers type that ends up being assignable from `[]byte` in Go, and in JavaScript can be obtained as a `Uint8Array`.

After #63, you can ask for the bytes to be a UTF16 string, making it easier to decode as a string.
